lib/linguist/languages.yml

```diff
@@ -2704,6 +2705,7 @@ Markdown:
   wrap: true
   extensions:
   - ".md"
+  - ".lr"
```

Comment:
You mean ``contents.lr``? Thanks for bringing this up.

I think so, but its a little strange. ``extension:lr`` got 15K hits, while  ``filename:contents.lr`` only got 12K hits. However, going through page after page of search results I couldn't find anything but ``contents.lr``, and despite several attempts I was not able to search for files with extension ``.lr`` that were NOT named contents.lr so I'm not sure what's going on with the other 3K. Are you able to glean further insight on this with your advanced knowledge and tools?

Reply:
Here they are: [`extension:lr -filename:contents.lr`](https://github.com/search?q=extension%3Alr+-filename%3Acontents.lr&type=Code). Looks like my typo wasn't so wrong after all :smile: 
Excluding `content.lr` files as well: [`extension:lr -filename:contents.lr -filename:content.lr`](https://github.com/search?p=5&q=extension%3Alr+-filename%3Acontents.lr+-filename%3Acontent.lr&type=Code).
Maybe we should add `contents.lr` as a Markdown filename rather than a new extension? I'll check to see if there are enough `content.lr` files to add it as well.

Reply:
> Maybe we should add contents.lr as a Markdown filename rather than a new extension?

If there's a way to easily do that, then that would probably be for the best since it would avoid the vast majority of false positives.

> I'll check to see if there are enough content.lr files to add it as well.

Looks like only 5 other people made your same typo across as many repos and 13 total files, so I'd think not :)

Reply:
Done, thanks!